Tests/SonarQube.MSBuild.Tasks.IntegrationTests/SonarQube.MSBuild.Tasks.IntegrationTests.csproj

```diff
@@ -57,25 +57,37 @@
     <Otherwise>
       <ItemGroup>
         <Reference Include="Microsoft.Build, Version=15.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <HintPath>..\..\packages\Microsoft.Build.15.1.548\lib\net46\Microsoft.Build.dll</HintPath>
+          <HintPath>..\..\packages\Microsoft.Build.15.5.180\lib\net46\Microsoft.Build.dll</HintPath>
```

Comment:
Needed to upgrade because of "Method '`[MSBuild]::IsRunningFromVisualStudio`' not found" [bug](https://github.com/Microsoft/msbuild/issues/2775).